src/accordion-item/index.js

```diff
@@ -0,0 +1,101 @@
+/**
+ * External dependencies
+ */
+import React from 'react'
+import { components, editor, element, i18n } from 'wp'
+/**
+ * Internal dependencies
+ */
+import './style.scss'
+
+
+const { Fragment } = element
+const { __ } = i18n
+
+const { PanelBody, ToggleControl } = components
+const { BlockControls, RichText, AlignmentToolbar, InnerBlocks, InspectorControls } = editor
+
+export const name = 'accordion-item'
+
+const ALLOWED_BLOCKS = [ 'core/image', 'core/paragraph', 'coliquio/image' ];
```

Comment:
Maybe not `core/image`? Please clarify with Vitali (iOS app)

Reply:
_or better:_ allow everything

Reply:
fixed